### ISSUE TITLE
Question about loss in the code

### ISSUE THREAD

**Issue by shidanW** [Jiahao000/MFM#5]
in paper it said that "In practice, we compute the loss only on the masked area of the frequency spectrum instead of the
full spectrum as the latter tends to decrease the accuracy according to our experiments."

however, in code i found that “loss = (loss_recon * (1 - mask.unsqueeze(1))).sum() / (1 - mask).sum() / self.in_chans / loss_recon.shape[1]”

it's actually contradictory, could you explain it? thank you
